concepts/05 UI Components/Toast/00 Getting Started with Toast/10 Customize Toast Content.md

```diff
@@ -0,0 +1,277 @@
+To customize toast content, either specify a [contentTemplate](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#contentTemplate) function or a custom template inside a component. Use `.dx-custom-toast` CSS class for the template and set the [type](/Documentation/ApiReference/UI_Components/dxToast/Configuration/#type) property to `custom`.
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        // ...
+        $("#custom").dxButton({
+            text: "Show custom message",
+            onClick: function() {
+                DevExpress.ui.notify(
+                    {
+                        width: 230,
+                        height: 50,
+                        position: {
+                            my: "bottom",
+                            at: "bottom",
+                            of: "#container"
+                        },
+                        contentTemplate: function () {
+                            return $("<p />")
+                                .text("You have a new message ")
+                                .addClass(".dx-custom-toast")
```

Comment:
does it really work? Why do u add a custom class on the paragraph in content? DevExtreme Toast is rendered with `dx-toast-custom` class on its content if type is specified as `custom`. I think you can use it here as you do in another approaches.

I also think that css styles can be the same in all approaches